### DISCUSSION TITLE
Swipe on video/long press to mark videos as 'not interested'

### DISCUSSION THREAD

**Discussion by stackingflow** [ReVanced/revanced-discussions#130] [Question]
Everyone knows youtube recommendations system is not perfect that's the reason it keeps on showing us bunch of random videos in our home feed, these are stuck in our feed until we watch them or mark them as `not interested` . Sometimes youtube keeps recommending us the video we have already watched. Only way to get rid of them is to mark them as `not interested`. Issue is `not interested` button is hidden deep inside the 3 dot menu, so the current process looks like this `Finds an uninteresting video (on home tab) > click 3 dot menu > click not interested`. Current process is very cumbersome as it included multiple clicks to perform a single task. We can improve our experience so much by just fine tuning our feeds. 

How we can achieve it?
1. Swipe right or left on video to automatically mark it as uninterested
2. Long press/hold on video to mark it as uninterested

Kind regards

**Comment by oSumAtrIX** [comment]
A gesture of this importance on thumbnail views should be reserved for something more important. Using the menu is fine.

**Comment by stackingflow** [comment]
This could be an optional patch (so that if people need it they can use it) not by default
